app/src/main/java/com/coopra/nebulus/views/adapters/AlternativeAdapter.java

```diff
@@ -0,0 +1,79 @@
+package com.coopra.nebulus.views.adapters;
+
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.coopra.data.Track;
+import com.coopra.nebulus.R;
+
+import java.util.List;
+import java.util.Locale;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+public class AlternativeAdapter extends RecyclerView.Adapter<AlternativeAdapter.MyViewHolder> {
+    private List<Track> mTracks;
+
+    public void setTracks(List<Track> tracks) {
+        mTracks = tracks;
```

Comment:
I wouldn't recommend setting this list directly to the adapter this way. This can lead to data inconsistencies and potentially cause a crash if the data is updated in the list without `notifyDataSetChanged()` being called. 

The best approach here would be to have `mTracks` be its own list, and in this method do something like this:

```suggestion
        mTracks.clear();
        mTracks.addAll(tracks);
```